stream_alert_cli/athena/handler.py

```diff
@@ -309,22 +297,23 @@ def athena_handler(options, config):
         options (namedtuple): The parsed args passed from the CLI
         config (CLIConfig): Loaded StreamAlert CLI
     """
-    athena_client = StreamAlertAthenaClient(config, results_key_prefix='stream_alert_cli')
```

Comment:
What's the benefit of removing the instantiation here and doing it in each subcommand's method?

Reply:
This allows any caller using these functions to not have to worry about created an athena client that needs to be passed it.

Example here:
https://github.com/airbnb/streamalert/blob/8d653cb2e423d00fcc640b780fa3675c0f05821f/stream_alert_cli/terraform/handler.py#L121

Also, not every subcommand needs this (ie: `init`)